### PR TITLE
Refactor to C# naming standards

### PR DESCRIPTION
**Pull request by kennyvv**
As discussed with @gdkchan

### CODE REVIEW

**Comment by emmauss** [comment]
rebase please

**Comment by kennyvv** [comment]
i've merged the changes.

**Comment by gdkchan** [comment]
Build is failing.

**Comment by kennyvv** [comment]
There you go buddy! @gdkchan

File: Ryujinx.Graphics/Gal/OpenGL/OGLExtension.cs [hunk 1]
```diff
@@ -3,23 +3,23 @@
 
 namespace Ryujinx.Graphics.Gal.OpenGL
 {
-    static class OGLExtension
+    static class OglExtension
     {
-        private static Lazy<bool> s_EnhancedLayouts    = new Lazy<bool>(() => HasExtension("GL_ARB_enhanced_layouts"));
-        private static Lazy<bool> s_TextureMirrorClamp = new Lazy<bool>(() => HasExtension("GL_EXT_texture_mirror_clamp"));
-        private static Lazy<bool> s_ViewportArray      = new Lazy<bool>(() => HasExtension("GL_ARB_viewport_array"));
+        private static Lazy<bool> _sEnhancedLayouts    = new Lazy<bool>(() => HasExtension("GL_ARB_enhanced_layouts"));
```
[the remark this answers is not in the document]
Comment (jduncanator) [reply Ryujinx.Graphics/Gal/OpenGL/OGLExtension.cs]: In old conventions perhaps, however I don't believe this is used very often anymore (at least not in any of my code bases at work, and not throughout the .NET source code). Usually fields are named: `_lowerCase` and private properties are `_UpperCase`, although YMMV depending on which code base you look at.

File: Ryujinx.Graphics/Texture/ImageUtils.cs [hunk 2]
```diff
@@ -42,93 +42,93 @@ public ImageDescriptor(int BytesPerPixel, int BlockWidth, int BlockHeight, Targe
         private const GalImageFormat Float = GalImageFormat.Float;
         private const GalImageFormat Srgb  = GalImageFormat.Srgb;
 
-        private static readonly Dictionary<GalTextureFormat, GalImageFormat> s_TextureTable =
+        private static readonly Dictionary<GalTextureFormat, GalImageFormat> STextureTable =
```
[the remark this answers is not in the document]
Comment (kennyvv) [reply Ryujinx.Graphics/Texture/ImageUtils.cs]: Its a static readonly variable, those don't have underscores ;)

[the remark this answers is not in the document]
Comment (jduncanator) [reply Ryujinx.Graphics/Texture/ImageUtils.cs]: They do in .NET source code, and in projects I've worked on. It's a field, it should have an underscore regardless of if it's readonly/static.

File: Ryujinx.Graphics/Gal/OpenGL/OGLPipeline.cs [hunk 3]
```diff
@@ -90,50 +90,51 @@ public OglPipeline(OglConstBuffer buffer, OglRasterizer rasterizer, OglShader sh
...
-                DepthWriteEnabled = true,
-                DepthFunc = GalComparisonOp.Less,
+		        CullFaceEnabled = false,
+		        CullFace        = GalCullFace.Back,
 
+		        DepthTestEnabled  = false,
+		        DepthWriteEnabled = true,
+		        DepthFunc         = GalComparisonOp.Less,
+		        DepthRangeNear    = 0,
+				DepthRangeFar     = 1,
```
Comment (gdkchan) [inline Ryujinx.Graphics/Gal/OpenGL/OGLPipeline.cs]: Identation is off here.

**Comment by gdkchan** [comment]
Not all alignment issues I poninted out were fixed. And it still needs a proper rebase, as it is now, it undoes the last commit on master, thus losing fixes.

**Pull request closed without merging**